packages/@okta/vuepress-site/.vuepress/public/docs/api/postman/myaccount.json

```diff
@@ -0,0 +1,901 @@
+{
+	"info": {
+		"_postman_id": "d4a57497-b5ce-4bd2-b4a8-040413e3c2ef",
+		"name": "MyAccount (Okta API)",
+		"description": "The [Okta MyAccount API](https://developer.okta.com/docs/reference/api/myaccount/) provides operations to allow end users (with or without administrator access) in your organization to fetch and update their own Okta user profiles.",
```

Comment:
we should remove '(with or without administrator access)'. for update, myaccount api only supports non-admin.